make_timeline.py

```diff
@@ -6,103 +6,263 @@
 This plot features the predicted ACIS attenuated fluence based on current ACIS orbital
 fluence, 2hr average, and grating status.  It also shows DSN comms, radiation zone
 passages, instrument configuration.
+
+Testing
+=======
+First some setup which applies to testing both on HEAD and local::
+
+  cd ~/git/arc
+  rm -rf t_now
+  COMMIT=`git rev-parse --short HEAD`
+  PR=pr86  # or whatever
+  mkdir -p t_now/$COMMIT
+  mkdir -p t_now/flight
+
+HEAD
+-----
+Testing on HEAD does an apples-to-apples comparison of the branch to the current flight
+code, running both in "flight" mode::
+
+  # Current flight version of make_timeline.py looks for data in the output directory.
+  ln -s /proj/sot/ska/data/arc3/*.h5 t_now/flight/
+  ln -s /proj/sot/ska/data/arc3/ACE_hourly_avg.npy t_now/flight/
+
+  # Run PR branch version in "flight" mode (no --test), being explicit about data
+  # directory. This version looks for data resources in fixed locations, not in
+  # `data-dir`, so starting with an empty directory is fine.
+  python make_timeline.py --data-dir=t_now/$COMMIT
+
+  # Run flight version of make_timeline.py in a directory hidden from the git repo.
+  cd t_now
+  python /proj/sot/ska/share/arc3/make_timeline.py --data-dir=flight
+
+  cd ..
+  rsync -av t_now/ /proj/sot/ska/www/ASPECT_ICXC/test_review_outputs/arc/${PR}/t_now/
+
+  diff t_now/{flight,$COMMIT}/timeline_states.js
```

Comment:
I didn't fine this diff very useful - and then tried jsondiff or to pretty-format the json file, but can't read it with json.load because it isn't actually a json file?  Maybe making it a json file would make sense as an infrastructure fix in this PR?

Reply:
For now, I used the convert script you provided to get to that yaml diff you found more useful for local testing.

Reply:
And from a new look at the js file, I guess that's a javascript file with an object literal json thing.  I guess I was just hoping that there was some convenient javascript read thing that could be done in the loading in the browser so the file would be easier to read.  But maybe that's not easy.

Reply:
Yeah, I should maybe just have skipped diffing the javascript directly, except that I think when I did it there was no diff. In any case the YAML serialization below is what we want if there is a diff.